types/types_test.go

```diff
@@ -216,8 +216,7 @@ func TestWithAttachmentsIncludingNestedCoreFiles(t *testing.T) {
 
 func TestValidateBrokenMetadata(t *testing.T) {
 	r := strings.NewReader(`#version: 0.1.0-missing
-name: _INVALID-name
-namespace: myhubusername
+name: MustBeAValidUrlButNotEvaluatedByTheSchema
```

Comment:
Not a Url, but an untagged registry reference

Reply:
Renamed